### PR TITLE
Appkeeper Integration

### PR DESCRIPTION
**Pull request by numno1**
### What does this PR do?

Adds SIOS AppKeeper integration to integrations-extras.

### Motivation

To  newly create AppKeeper Dashboard.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
Only non-code files.
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)
no.

### CODE REVIEW

File: appkeeper/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Agent Check: appkeeper
+
+## Overview
+
+SIOS AppKeeper can automatically restart failed Amazon EC2 services when
```
Comment (sarina-dd) [inline appkeeper/README.md:5]:
```suggestion
SIOS AppKeeper automatically restarts failed Amazon EC2 services when
```

File: appkeeper/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+notifications are received from Datadog such as Synthetics, removing
+the need for expensive manual intervention.
+
+When Datadog finds the alerts, it can restart the Service via AppKeeper Recovery API.
```
Comment (sarina-dd) [inline appkeeper/README.md:9]:
```suggestion
When Datadog finds the alerts, it restarts the Service via AppKeeper Recovery API.
```

File: appkeeper/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Integration between Webhook notification with Datadog and SIOS AppKeeper Recovery API
```
Comment (sarina-dd) [inline appkeeper/README.md:11]: Is this sentence still a WIP?

File: appkeeper/README.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+
```
Comment (sarina-dd) [inline appkeeper/README.md:13]:
```suggestion
```

File: appkeeper/README.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+## Setup
+
+### Step 1. Get the SIOS AppKeeper API Key
+
+Get the SIOS AppKeeper API Key from AppKeeper GUI.
+
+![snapshot][2]
+
+### Step 2. Define the Webhook in the Datadog Integration Dashboard.
```
Comment (sarina-dd) [inline appkeeper/README.md:22]:
```suggestion
### Step 2. Define the Webhook in the Datadog Integration Dashboard
```

File: appkeeper/README.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+![snapshot][3]
+
+### Step 3. Define the PAYLOAD and CUSTOM HEADERS.
```
Comment (sarina-dd) [inline appkeeper/README.md:26]:
```suggestion
### Step 3. Define the PAYLOAD and CUSTOM HEADERS
```

File: appkeeper/README.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+![snapshot][4]
+
+1. Enter the **URL** "https://api.appkeeper.sios.com/v2/integration/{AWS_account_ID}/actions/recover"
```
Comment (sarina-dd) [inline appkeeper/README.md:30]:
```suggestion
1. Enter the **URL**: "https://api.appkeeper.sios.com/v2/integration/{AWS_account_ID}/actions/recover"
```

File: appkeeper/README.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+1. Enter the Instance ID and name of Services for the monitoring instance in the **Payload**
```
Comment (sarina-dd) [inline appkeeper/README.md:31]:
```suggestion
2. Enter the Instance ID and name of Services for the monitoring instance in the **Payload**
```

File: appkeeper/README.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+1. Register the AppKeeper API token in the **Custom Headers** "**appkeeper-integration-token**"
```
Comment (sarina-dd) [inline appkeeper/README.md:32]:
```suggestion
3. Register the AppKeeper API token in the **Custom Headers** "**appkeeper-integration-token**"
```

File: appkeeper/README.md [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+For more information on AppKeeper's Integration, please see below.
```
Comment (sarina-dd) [inline appkeeper/README.md:34]:
```suggestion
For more information on the AppKeeper Integration, please review the Appkeeper [documentation][5].
```

File: appkeeper/README.md [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+https://sioscoati.zendesk.com/hc/en-us/articles/900000978443-Integration
```
Comment (sarina-dd) [inline appkeeper/README.md:36]:
```suggestion
```

File: appkeeper/README.md [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+### Metrics
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][1].
+
+[1]: https://docs.datadoghq.com/help/
+[2]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/appkeeper/images/get_token2.png
+[3]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/appkeeper/images/datadog_webhook.jpg
+[4]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/appkeeper/images/payload_header.jpg
```
Comment (sarina-dd) [inline appkeeper/README.md:49]:
```suggestion
[4]: https://raw.githubusercontent.com/DataDog/integrations-extras/master/appkeeper/images/payload_header.jpg
[5]: https://sioscoati.zendesk.com/hc/en-us/articles/900000978443-Integration
```

File: appkeeper/manifest.json [hunk 13]
```diff
@@ -0,0 +1,32 @@
+{
+  "display_name": "AppKeeper",
+  "maintainer": "rd-pd-1@sios.com",
+  "manifest_version": "1.0.0",
+  "name": "AppKeeper",
+  "metric_prefix": "AppKeeper.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Appkeeper restarts service based on alarms from Datadog",
```
Comment (sarina-dd) [inline appkeeper/manifest.json:9]:
```suggestion
  "short_description": "Appkeeper restarts service based on alerts from Datadog",
```

**Review by sarina-dd — CHANGES_REQUESTED** [review]
Editorial review completed

File: appkeeper/README.md (hunk 1 above)
Comment (numno1) [reply appkeeper/README.md]: 👍

File: appkeeper/README.md (hunk 2 above)
Comment (numno1) [reply appkeeper/README.md]: 👍

File: appkeeper/README.md (hunk 3 above)
Comment (numno1) [reply appkeeper/README.md]: 👍

File: appkeeper/README.md (hunk 4 above)
Comment (numno1) [reply appkeeper/README.md:10]: 👍

File: appkeeper/README.md (hunk 5 above)
Comment (numno1) [reply appkeeper/README.md]: 👍

File: appkeeper/README.md (hunk 9 above)
Comment (numno1) [reply appkeeper/README.md]: 👍

File: appkeeper/README.md (hunk 10 above)
Comment (numno1) [reply appkeeper/README.md]: 👍

File: appkeeper/README.md (hunk 11 above)
Comment (numno1) [reply appkeeper/README.md]: 👍

File: appkeeper/README.md (hunk 12 above)
Comment (numno1) [reply appkeeper/README.md:44]: 👍

File: appkeeper/manifest.json (hunk 13 above)
Comment (numno1) [reply appkeeper/manifest.json]: 👍

**Comment by numno1** [comment]
Thank you for your reviewing.
I fixed the indicated items. Review again, please.

**Review by sarina-dd — APPROVED** [review]
Thanks for the contribution! Approved for docs :)

**Review by hithwen — APPROVED** [review]

**Pull request merged**